projects/plugins/jetpack/class.jetpack-gutenberg.php

```diff
@@ -7,6 +7,7 @@
  */
 
 use Automattic\Jetpack\Blocks;
+use Automattic\Jetpack\Connection\Client;
```

Comment:
I think we can remove this?

Reply:
Thanks, Harris. Addressed here decfd15306456c65839a4625116bef6d51abdb5d